dashboard/app/views/pd/workshop_mailer/_teacher_enrollment_details.html.haml

```diff
@@ -60,12 +60,14 @@
 %p
   If you are unable to attend the workshop, please
   = link_to 'cancel your registration', @cancel_url
-  , in advance so the facilitator can plan accordingly.
+  in advance so the facilitator can plan accordingly.
 
 %p
   Thank you,
   %br
   = @workshop.organizer.name
 
 %p
-  P.S. Connect with other computer science teachers on Twitter @TeachCode for inspiration and great tips on bringing #CSforAll.
+  P.S. Connect with other computer science teachers on Twitter
+  = link_to '@TeachCode', "https://twitter.com/teachcode"
```

Comment:
nit: ruby style - use single-quote strings when not interpolating. See https://github.com/bbatsov/ruby-style-guide#consistent-string-literals